snap/quota/quota.go

```diff
@@ -240,13 +240,14 @@ type groupQuotaAllocations struct {
 	MemoryLimit    quantity.Size
 	MemoryReserved quantity.Size
```

Comment:
maybe for clarity these should be called *ReservedByChildren?

Reply:
I agree, this makes sense